app/assets/stylesheets/ama_layout/layout_components/callouts.scss

```diff
@@ -142,3 +142,22 @@
     }
   }
 }
+
+.callout-side-arrow{ //this works best if it's a class on a svg element.
+  position: absolute;
+  z-index: 1;
+  top: 0;
+
+  @include breakpoint(small down){
+    top: 100%;
+    left: 0%;
+    height: 30%;
+    background-color: $white;
+  }
+}
+
+.mobile-arrow-margin{
```

Comment:
Sorry, I meant the element below. Adding the margin to the arrow itself doesnt really work since it's positioned absolutely. The margin is needed so that the content in the content half below the image starts below the height of the arrow. I could add it to the parent of the arrow though, but that doesnt have any arrow specific classes either.